values-schema.yaml

```diff
@@ -2124,7 +2124,16 @@ properties:
                 $ref: '#/definitions/resources'
               reportsController:
                 $ref: '#/definitions/resources'
-
+      policy-reporter:
+        additionalProperties: false
+        properties:
```

Comment:
It is fine to be able to disable that app but the current implementation in this PR does not allow to do it.